pkg/kubelet/certificate/bootstrap/bootstrap.go

```diff
@@ -262,7 +262,7 @@ func isClientConfigStillValid(kubeconfigPath string) (bool, error) {
 	now := time.Now()
 	for _, cert := range certs {
 		if now.After(cert.NotAfter) {
-			utilruntime.HandleError(fmt.Errorf("part of the existing bootstrap client certificate is expired: %s", cert.NotAfter))
+			utilruntime.HandleError(fmt.Errorf("part of the existing bootstrap client certificate which located at %q is expired: %s", kubeconfigPath, cert.NotAfter))
```

Comment:
SGTM, suggesting a minor change:

```suggestion
			utilruntime.HandleError(fmt.Errorf("part of the existing bootstrap client certificate located at %q is expired: %v", kubeconfigPath, cert.NotAfter))
```